remoteworkitem/trackerquery_repository_test.go

```diff
@@ -45,23 +45,32 @@ func (test *TestTrackerQueryRepository) TestTrackerQueryCreate() {
 	params := url.Values{}
 	ctx := goa.NewContext(context.Background(), nil, req, params)
 
-	query, err := test.queryRepo.Create(ctx, "abc", "xyz", uuid.NewV4(), space.SystemSpace)
-	assert.IsType(t, remoteworkitem.InternalError{}, err)
-	assert.Nil(t, query)
+	testFxt := tf.NewTestFixture(test.T(), test.DB, tf.Spaces(1))
+
+	tq := remoteworkitem.TrackerQuery{
+		Query:     "abc",
+		Schedule:  "xyz",
+		TrackerID: uuid.NewV4(),
+		SpaceID:   testFxt.Spaces[0].ID,
+	}
+
+	err := test.queryRepo.Create(ctx, &tq)
+	assert.IsType(t, errors.InternalError{}, err)
 
 	tracker := remoteworkitem.Tracker{
 		URL:  "http://issues.jboss.com",
 		Type: remoteworkitem.ProviderJira,
 	}
 	err = test.trackerRepo.Create(ctx, &tracker)
-	query, err = test.queryRepo.Create(ctx, "abc", "xyz", tracker.ID, space.SystemSpace)
-	require.NoError(t, err)
-	assert.Equal(t, "abc", query.Query)
-	assert.Equal(t, "xyz", query.Schedule)
 
-	query2, err := test.queryRepo.Load(ctx, query.ID)
-	require.NoError(t, err)
-	assert.Equal(t, query, query2)
+	tq.TrackerID = tracker.ID
+
+	err = test.queryRepo.Create(ctx, &tq)
+	assert.Nil(t, err)
+
+	tq2, err := test.queryRepo.Load(ctx, tq.ID)
+	assert.Nil(t, err)
```

Comment:
Please use the already well established `require.NoError` here:
```suggestion
	require.NoError(t, err)
```

Reply:
corrected here 4cee394

Reply:
No, please see my comment https://github.com/fabric8-services/fabric8-wit/commit/4cee3948982fde26b55c6b0c2d82caa538e15c94#r31463970